utiliti/src/de/gurkenlabs/utiliti/handlers/Transform.java

```diff
@@ -201,9 +201,6 @@ public static void move() {
     // calculate the actual delta that all the map objects need to be moved by
     float deltaX = newX - drag.minX;
     float deltaY = newY - drag.minY;
-    if (deltaX == 0 && deltaY == 0) {
```

Comment:
Any particular reason, you removed this? If there is no delta in either direction there should also be no point in moving the entity at all which is why this originally just returned at that point.

Reply:
I should've commented on this, sorry.
I had a problem while draging objects. I could drag them anywhere, except back to where they originally were. Removing that condition solved the problem.